extensions/amp-mraid/0.1/test/validator-amp-mraid.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
```

Comment:
Please include the license and test description. You'll need to update the `.out` file as well.

```
<!--
  Copyright 2019 The AMP HTML Authors. All Rights Reserved.
  Licensed under the Apache License, Version 2.0 (the "License");
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at
      http://www.apache.org/licenses/LICENSE-2.0
  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS-IS" BASIS,
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the license.
-->
<!--
  Test Description:
  Tests for the amp-mraid tag. See the inline comments.
-->
```

Reply:
done